user-guide/modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,886 @@
+
+[source]
+----
+cmake --build . --target install --config Debug
+cmake --build . --target install --config Release
+----
+
+Although both the boost:build[] and the CMake scripts work fine, the final installed libraries will have different layouts for historical reasons.
+Both the CMake integration scripts and the libraries will be installed under different paths and this can create incompatibilities between Boost installations.
+
+TIP: You don't need to *build* Boost with CMake to be able to *use* Boost with CMake.
+CMake integration will work fine even if you install Boost with boost:build[].
+
+The supported and tested way to build Boost remains with boost:build[] and you should refrain from using CMake at this step unless you need this functionality.
+You can find more information about the limitations of a CMake infrastructure on https://github.com/boostorg/cmake[boostorg/cmake,window="_blank"]
+--
+====
+
+=== Environment variables
+
+We recommend you update your environment variables after installing Boost.
+When you update your environment variables, you are telling your operating system and other tools where to look for the Boost libraries and headers.
+
+[tabs,sync-group-id=os]
+====
+Windows::
++
+--
+
+[source]
+----
+set BOOST_ROOT=C:\boost <1>
+----
+
+<1> Specify the root directory of the Boost pass:[C++] libraries so other tools can find it
+
+IMPORTANT: Replace `C:\boost` with the prefix directory you specified during the installation, if different.
+
+--
+
+Linux::
++
+--
+
+[source,bash]
+----
+export BOOST_ROOT=/usr/local <1>
+export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH <2>
+export CPLUS_INCLUDE_PATH=/usr/local/include:$CPLUS_INCLUDE_PATH <3>
+----
+
+<1> Specify the root directory of the Boost pass:[C++] libraries so other tools can find it
+<2> Specify additional directories to search for shared libraries when executing a program
+<3> Specify additional directories to search for pass:[C++] header files
+
+IMPORTANT: Replace `/usr/local` with the prefix directory you specified during the installation, if different.
+
+You can add these `export` commands to your `~/.bashrc` or `~/.profile` file to make the changes permanent.
+
+--
+
+macOS::
++
+--
+
+[source,bash]
+----
+export BOOST_ROOT=/usr/local <1>
+----
+
+<1> Specify the root directory of the Boost pass:[C++] libraries so other tools can find it
+
+IMPORTANT: Replace `/usr/local` with the prefix directory you specified during the installation, if different.
+
+You can add these `export` commands to your `~/.bashrc` or `~/.profile` file to make the changes permanent.
+
+--
+====
+
+[#your-first-app]
+== Your First App
+
+=== Header-only libraries
+
+Let’s start by using a header-only library.
+Copy the following program into a file called `example.cpp`.
+
+[source,C++]
+.example.cpp
+----
+#include <boost/lambda/lambda.hpp> <1>
```

Comment:
I don't know if this PR is the place to put this comment - but I'd provide another example. This is joining lambda (seems an old lib, and I can hear users "but we already have this in the language already") with iostreams (which we all love).

Reply:
This is the example we have in the getting started page now. But I agree. I would open an issue for that because that requires some extra thinking about newer libraries we could use there. I think it would be nice to have something they feel they can only get from boost, like boost.unordered or something. Although boost.unordered doesn't have the same appeal to someone just getting started. Lambda2 might be a good option, although "2" in the first app doesn't look good to me. We definitely don't want something we have in the standard library. Well... it requires some thinking.